doc/modules/language-guide/pages/language-manual.adoc

```diff
@@ -45,6 +45,8 @@ For example, this section provides technical details of interest to the followin
 The language quick reference is intended to provide complete reference information about {proglang}, but this section does _not_ provide explanatory text or usage information.
 Therefore, this section is typically not suitable for readers who are new to programming languages or who are looking for a general introduction to using {proglang}.
 
+Throughout, we use the term canister to refer to an {IC} canister smart contract.
```

Comment:
This is in introduction, right?

It seems like we really want to be explicit and say "canister smart contract" intsead of "canister" -- there are a lot of changes in this PR for that. So maybe we can remove this, or just use "canister" everywhere after this paragraph?

Reply:
So the approach I'm taking is that the first mention on a page is canister smart contract, and then just canister for the rest of the page. Which I think was sanctioned elsewhere. To be honest, I can't give a toss anymore.